From 5b57fbda6d3b9d1aa56d0df74658e47bf9859552 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Fri, 30 May 2014 01:03:44 +0900
Subject: [PATCH] Remove clang -Winvalid-source-encoding warning

Remove clang -Winvalid-source-encoding warning as shown below:
../../driver/XScreenSaver_ad.h:298:33: warning: illegal character
  encoding in string literal [-Winvalid-source-encoding]
"*splash.body.label:            Copyright <A9> 1991-2014 by",

Just use hexadecimal expression instead of non-ascii character
to suppress this warning.
The result of "objdump -s demo-Gtk.o" does not change after this,
so this modification should actually does no change.
---
 driver/XScreenSaver.ad.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/driver/XScreenSaver.ad.in b/driver/XScreenSaver.ad.in
index dde6c8c..25e9c28 100644
--- a/driver/XScreenSaver.ad.in
+++ b/driver/XScreenSaver.ad.in
@@ -421,7 +421,7 @@ XScreenSaver.bourneShell:		/bin/sh
 *passwd.uname:                  True
 
 *splash.heading.label:		XScreenSaver %s
-*splash.body.label:		Copyright © 1991-2014 by
+*splash.body.label:		Copyright \xa9 1991-2014 by
 *splash.body2.label:		Jamie Zawinski <jwz@jwz.org>
 *splash.demo.label:		Settings
 *splash.help.label:		Help
-- 
1.9.3

